libs/device/duck/duckHurt.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Looks like you copied meta.json here instead of one of the individual sprite jsons. I don't think you need all of this extra info

Reply:
ooo yes amazing catch ty